### DISCUSSION TITLE
[Suggestion] Weapon type ideas

### DISCUSSION THREAD

**Discussion by thatisjake** [tsunamayo/Starship-EVO#4336] [Suggestions]
### Suggestion
**Swarm Missiles**
They would be a weapon brick, each having ~20 missiles. On logic activation, they all shoot and aim to the nearest hostile. They do probably do low hull damage, and very low shield damage. Would be effective against swarms of drones

**Tesla Coil**
Shoots a arc of electricity to the nearest hostile, if it hits one, it bounces to other hostiles nearby. Low shield damage, and view low hull damage. Possibly a small heat increase on the target.

**Point defense option on turrets**
A option toggleable on turrets, to automatically shoot at incoming missiles. Very useful if needing to defend against swarm missiles.

**Torpedo**

Torpedo that simply detach from a ship, and head straight forward. 
2 ways these could work:

The "custom built" method
This way would need 3 more blocks.
A "detach" brick, which on logic separates the missile from the ship. 
 A warhead block, when primed & on contact, explodes. Medium hull damage, low shield damage.
 A Missile thruster block, when given signal, it thrusts forward. (even when detached from a ship).
 
The "Chip" method
A default torpedo, with addon slots, similar to laser or beam weapons.

**Guided Missiles**
This could work the same way as torpedoes,  but with a "missile computer". Simply put, it would define which target to aim at.

**EMP**
This could work in a couple ways
Custom way: It could be a block, when on signal temporarily drains power from nearby ships. If put onto a custom built missile, you can create EMP missiles.

Easy way: A simple gun that shoots out a EMP shot. On contact with a ship, drains some power.

**Comment by tsunamayo** [comment]
Hi, yep I wanted to do craftable missiles. Thanks
